pages/authentification.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import imaplib
+import email
+
+import time
+from tempmail import TempMail
+from selenium.webdriver.common.by import By
+
+from pages.base import Base
+
+
+class Authentification(Base):
```

Comment:
I'd rename this file to be `auth_zero.py` and the class to be `AuthZero`, and I don't think it should extend from `Base` as it's a page from another party.